ChangeLog.md

```diff
@@ -239,6 +243,8 @@
 
   * Fix gradebook and question statistics download links (Tim Bretl).
 
+  * Fix `demoRandomPlot` by updating the matlibplot subplot code (James Balamuta).
+
```

Comment:
We are actually not bothering with ChangeLog entries anymore. The announcements are generated from the git log. At some point (after the next week) I will set it up so that we auto-dump the git log into the ChangeLog file.